source/content/drupal-updates.md

```diff
@@ -16,27 +16,27 @@ reviewed: "2020-07-21"
 
 The steps in this doc help bring your site to Drupal 8 which reached End of Life status in November, 2021.
 
-Visit the [Drupal 9 migration guide](/drupal-9-migration) to find the best path to Drupal 9 for your site.
+Visit the [Drupal Update and Migration Guide](/drupal-migration) to find the best path to Drupal for your site.
 
 </Alert>
 
 ## Overview
 
-To upgrade Drupal to a new major version (e.g. version 7 to version 9) you must create a new site. Do not perform a major version upgrade from within the original site. If you have a Drupal 7 site that you want to upgrade to Drupal 9, create a new Drupal 9 site and add content, files and modules from the old site into the new site.
+To upgrade Drupal to a new major version (e.g. version 7 to version 9) you must create a new site. Do not perform a major version upgrade from within the original site. If you have a Drupal site that you want to upgrade to the latest version of Drupal, create a new Drupal site and add content, files and modules from the old site into the new site.
```

Comment:
We should not remove the "7"s here. Drupal 8 represented a major architectural change to the way sites are built. This advice that you must create a new site only applies to Drupal 7 sites upgrading to >= Drupal 8.

Reply:
done